.github/workflows/release-please.yml

```diff
@@ -0,0 +1,18 @@
+name: Release Please
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
does this mean release on every main pushing? will be too many I guess, manual release or on some special tag seems better I think.

Reply:
no this means that the workflow will run on main, but the workflow is only to manage the release-please branch (rebase it against main, and force-push an up-to-date changelog).

only when this PR is merged will a release be created on github with a tag.